### DISCUSSION TITLE
Can we possibly get Good Ol' Winamp colors?

### DISCUSSION THREAD

**Discussion by abd11a** [monkeytypegame/monkeytype#1846] [Ideas]
I've got this so far:
Dark Purple: #292939
Lighter Purple: #474767
Green Highlight: #01F301

There's also a pale yellow in the Window title.

**Comment by ShahriarKh** [comment]
You can add your custom theme:
https://github.com/Miodec/monkeytype/blob/dev/THEMES.md
